LA notes.tex

```diff
@@ -311,7 +311,7 @@ \section{Solving Linear Systems}
 
 For some properties of the system of equations, we need only look at the coefficient matrix but others will need the augmented matrix. It is important to know the difference and be able to state which corresponding matrix you are using in your work.
 
```

Comment:
Question 10 does not include linear systems, which confused the students. Also, the given list would have 11 answers, not 10, as stated. Maybe this was meant to be a teaching point, like "we can't do what's asked for question 10 because it's not linear systems" but I didn't find it effective or implement it well. If you keep it, I suggest adding an instructor note.